app/src/main/res/layout/item_place.xml

```diff
@@ -6,13 +6,22 @@
     android:focusableInTouchMode="true"
     android:minHeight="@dimen/large_height">
 
+    <ImageView
+      android:id="@+id/bookmarkButtonImage"
+      android:layout_width="wrap_content"
+      android:layout_height="wrap_content"
+      android:layout_margin="@dimen/standard_gap"
+      android:tint="?attr/rowButtonColor"
+      app:srcCompat="@drawable/ic_round_star_border_24px" />
+
     <com.facebook.drawee.view.SimpleDraweeView
         android:id="@+id/icon"
         android:layout_width="@dimen/dimen_40"
         android:layout_height="@dimen/dimen_40"
         android:layout_marginLeft="@dimen/standard_gap"
         android:layout_marginStart="@dimen/standard_gap"
         android:layout_marginTop="@dimen/standard_gap"
+        android:layout_alignLeft="@id/bookmarkButtonImage"
```

Comment:
Is Left what we want? Start is generally what should be used, also these other attributes are confused, you shouldn't specify left and start

![image](https://user-images.githubusercontent.com/3358282/82038402-65c92600-969b-11ea-9159-79e00fa7d7d0.png)